crypto_hasher.go

```diff
@@ -20,6 +20,8 @@ import (
 	"crypto/sha1"
 	"crypto/sha256"
 	"crypto/sha512"
+	"github.com/amzn/ion-hash-go/ihp"
+	"github.com/amzn/ion-hash-go/internal"
```

Comment:
In previous PRs I was told to put the github ion-go import at the bottom separated from the other imports. Should we do the same thing with ihp and internal?

Reply:
Yeah this should have been caught by the CI's task for goimports. I might have to update the config to go into subfolders.

Reply:
See #33

Reply:
Updated